code/modules/clothing/gloves/color.dm

```diff
@@ -80,6 +84,17 @@
 	. = ..()
 	siemens_coefficient = pick(0,0,0,0.5,0.5,0.5,0.75)
 
+/obj/item/clothing/gloves/cut
+	desc = "These gloves would protect the wearer from electric shock... if the fingers were covered."
+	name = "fingerless insulated gloves"
+	icon_state = "yellowcut"
+	inhand_icon_state = "ygloves"
+	transfer_prints = TRUE
+	cut_type = TRUE
```

Comment:
```suggestion
```
`cut_type` is not a boolean variable, and you don't want to be able to cut already-cut gloves, do you?